### PR TITLE
Add get single derived key endpoint and update testnet eth url

### PR DESCRIPTION
**Pull request by lazynina**

### CODE REVIEW

File: routes/user.go [hunk 1]
```diff
@@ -3263,6 +3249,80 @@ func (fes *APIServer) GetUserDerivedKeys(ww http.ResponseWriter, req *http.Reque
...
+type GetSingleDerivedKeyResponse struct {
+	DerivedKey *UserDerivedKey
+}
+
+func (fes *APIServer) GetSingleDerivedKey(ww http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+
+	ownerPublicKeyBase58Check, ownerPublicKeyExists := vars["ownerPublicKeyBase58Check"]
+	if !ownerPublicKeyExists {
+		_AddBadRequestError(ww, fmt.Sprintf("GetSingleDerivedKey: ownerPublicKeyBase58Check required"))
```
Comment (DeSoDog) [inline routes/user.go:3261]: Add a return statement here

File: routes/server.go [hunk 2]
```diff
@@ -1044,6 +1045,13 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.GetUserDerivedKeys,
 			PublicAccess,
 		},
+		{
+			"GetSingleDerivedKey",
+			[]string{"GET"},
+			RoutePathGetSingleDerivedKey + "/{ownerPublicKeyBase58Check:[0-9a-zA-Z]{54,55}}/{derivedPublicKeyBase58Check:[0-9a-zA-Z]{54,55}}",
```
Comment (DeSoDog) [inline routes/server.go:1051]: What does the regex mean here? Is that a type of validation guard?

**Review by DeSoDog — CHANGES_REQUESTED** [review]

File: routes/user.go (hunk 1 above)
Comment (lazynina) [reply routes/user.go:3261]: good catch. thanks!

File: routes/server.go (hunk 2 above)
Comment (lazynina) [reply routes/server.go:1051]: it just ensures that we get a valid public key at this route param

**Pull request merged**